mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -652,6 +653,11 @@ be from 0 to 999,999,999 inclusive.`,
 								},
 							},
 						},
+						"maintenance_interval" : {
```

Comment:
I checked the code, [expandScheduling](https://github.com/GoogleCloudPlatform/magic-modules/blob/1bca5c3c8d1874eb95d6ed2a7a53cbef83b32f58/mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb#L64) is used to [expand scheduling](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb#L1692). The new field is not included in this function. However, the function expandScheduling is shared by other resources. We do not want to break other resources. If add the new field to this function, we need to test other resources.

[flattenScheduling function](https://github.com/GoogleCloudPlatform/magic-modules/blob/1bca5c3c8d1874eb95d6ed2a7a53cbef83b32f58/mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb#L186) is similar.

Reply:
You are right, scheduling is shared between InstanceTemplate and Instance. 

I added changes to Instance resource.